agent/Makefile

```diff
@@ -254,7 +254,7 @@ install-python3-setup: install-util-scripts install-lib
 	${COPY} requirements.txt ${DESTDIR}
 	mkdir -p ${DESTDIR}/python3
 	(cd ..; SKIP_GENERATE_AUTHORS=1 SKIP_WRITE_GIT_CHANGELOG=1 python3 setup.py install --prefix=${DESTDIR}/python3)
-	${COPY} $(addprefix ${DESTDIR}/python3/bin/, ${click-scripts}) ${UTILDIR}/
+	${COPY} $(addprefix ${DESTDIR}/python3/local/bin/, ${click-scripts}) ${UTILDIR}/
```

Comment:
If I'm not mistaken, this was exactly what Riya tried to do for Fedora-36, but it doesn't work elsewhere, including on our Jenkins VMs:

```
cp: cannot stat '/var/tmp/jenkins/tox/py/src/agent/python3/local/bin/pbench-generate-token': No such file or directory
cp: cannot stat '/var/tmp/jenkins/tox/py/src/agent/python3/local/bin/pbench-cleanup': No such file or directory
```

Reply:
Yeah, this is why I posted this in a separate commit.  I needed to ensure an RPM built before submitting this.